src/api/v1beta1/properties.go

```diff
@@ -32,7 +33,7 @@ const (
 	PullSecretSuffix             = "-pull-secret"
 	ActiveGateTenantSecretSuffix = "-activegate-tenant-secret"
 	OneAgentTenantSecretSuffix   = "-oneagent-tenant-secret"
-	AuthTokenSecretSuffix        = "-activegate-authtoken-secret"
+	AuthTokenSecretSuffix        = "-activegate-authtoken-secret" //nolint:gosec
```

Comment:
I see many `//nolint:gosec`, does it provide enough value for the headache it provides ?

Reply:
Agree, I disabled `G101: Look for hard coded credentials` `gosec`'s rule and it looks way better now.
This rule matches variable names with `"pattern": "(?i)passwd|pass|password|pwd|secret|private_key|token",` which is fine for us to ignore it because it's typical k8s nomenclature.